scripts/generate_schema_docs/template.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"text/template"
+)
+
+var tmpl = template.Must(template.New("").Funcs(template.FuncMap{"lstrip": lstrip}).Parse(`# {{.Title}}
```

Comment:
I feel that this should just be in the same file as `lstrip`.

Reply:
This was not needed anymore, removed